index.js

```diff
@@ -0,0 +1,4 @@
+// ФункцЫя суммы
```

Comment:
Опечатка в строке! можно поправить 

```suggestion
// Функция суммы
```